### PR TITLE
DumpMaps() add optional list of maps to dump

### PR DESCRIPTION
**Pull request by nplanel**
DumpMaps(maps .…string)
empty list = dump all maps

* Remove PerfMapStats as it's not so much relevant/important

### CODE REVIEW

File: manager/manager.go [hunk 1]
```diff
@@ -247,27 +247,38 @@ type Manager struct {
 }
 
 // DumpMaps - Return a string containing human readable info about eBPF maps
-func (m *Manager) DumpMaps() (string, error) {
+// is not maps list given all maps with an DumpHandler set are dumped
```
Comment (brycekahle) [inline manager/manager.go:250]: Maybe reword to "Dumps the set of maps provided, otherwise dumping all maps with a DumpHandler set."

**Comment by brycekahle** [comment]
Let's split out the perf map removal to a separate PR. It makes this PR hard to review and the history confusing.

File: manager/manager.go (hunk 1 above)
Comment (nplanel) [reply manager/manager.go]: done

**Comment by nplanel** [comment]
> Let's split out the perf map removal to a separate PR. It makes this PR hard to review and the history confusing.

done

**Review by brycekahle — APPROVED** [review]
